chemicaltoolbox/rdock/rbcavity.xml

```diff
@@ -0,0 +1,90 @@
+<tool id="rbcavity" name="rDock cavity definition" version="0.1">
+    <description>- generate the active site definition needed for rDock docking</description>
+    <requirements>
+        <requirement type="package" version="2013.1-0">rdock</requirement>
+    </requirements>
+    <command><![CDATA[
+        ln -s $receptor_prm ./receptor.prm &&
+        ln -s '$receptor' receptor.mol2 &&
+        ln -s '$ligand' ligand.sdf &&
+        rbcavity -was -d -r receptor.prm &&
+        mv receptor.as '$activesite'
+    ]]></command>
+
+    <configfiles>
+        <configfile name="receptor_prm">RBT_PARAMETER_FILE_V1.00
+RECEPTOR_FILE receptor.mol2
+RECEPTOR_FLEX 3.0
+SECTION MAPPER
+    SITE_MAPPER RbtLigandSiteMapper
+    REF_MOL ligand.sdf
+    RADIUS $radius
+    SMALL_SPHERE $sphere
+    MIN_VOLUME $min_volume
+    MAX_CAVITIES 1
+    VOL_INCR $vol_incr
+    GRIDSTEP $gridstep
+END_SECTION
+SECTION CAVITY
+    SCORING_FUNCTION RbtCavityGridSF
+    WEIGHT $weight
+END_SECTION
+        </configfile>
+    </configfiles>
+
+    <inputs>
+        <param type="data" name="receptor" format="mol2" label="Receptor" help="Select a receptor (Mol2 format)."/>
+        <param type="data" name="ligand" format="mol,sdf" label="Reference ligand" help="Single ligand in Mol or SDF format"/>
+        <param name="radius" type="float" value="6.0" label="Mapper sphere radius" help="Mapper radius of large sphere in angstroms; rDock will search for cavities to the small sphere but not to the large sphere"/>
```

Comment:
we can add min here to all params isn't it?

Reply:
@bgruening Not sure what you're meaning here.

Reply:
A minimum value, probably `min="0"`.

Reply:
Added: https://github.com/tdudgeon/galaxytools/commit/23d7d698390b58f2bf9487a7ee8e011511ecb74c